src/main/java/com/couchbase/lite/store/SQLiteStore.java

```diff
@@ -351,6 +354,125 @@ private void decrypt(SymmetricKey encryptionKey) throws CouchbaseLiteException {
     }
 
     ///////////////////////////////////////////////////////////////////////////
+    // ENCRYPTABLE STORE
+    ///////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void setEncryptionKey(SymmetricKey key) {
+        encryptionKey = key;
+    }
+
+    @Override
+    public Action actionToChangeEncryptionKey(final SymmetricKey newKey) {
```

Comment:
Discussion Item: It seems actionToChangeEncryptionKey() can work only with SQLCipher. Not with default SQLite. Should it be implemented outside of SQLiteStore?

I am not against current structure. Just I'd like to have cleaner isolation if possible.